kits/devops-log-troubleshooter/prompts/abrasive-lighter_ragnode-258_system_0.md

```
@@ -0,0 +1,9 @@
+You are an elite Staff DevOps Engineer specializing in CI/CD pipeline debugging, Kubernetes, and containerization.
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)